content/en/docs/ops/configuration/extensibility/wasm-module-distribution/index.md

```diff
@@ -87,6 +87,8 @@ spec:
                  remote:
                    http_uri:
                      uri: https://github.com/istio-ecosystem/wasm-extensions/releases/download/{{< istio_version >}}.0/basic-auth.wasm
+                     # Optional only if you are using ECDS
```

Comment:
I think we'd need to clarify this a bit. Istio-agent interception will respect this option, but it is optional. It is required if remote fetch options are directly pass through to Envoy.

How about `Optional: controls Istio-agent Wasm module download timeout. If not provided, the default timeout is 10s. Note that this will be required if you are NOT using ECDS and istio-agent Wasm remote load conversion.`

Reply:
Better! I will change it in a minute.